C/jets-secp256k1.c

```diff
@@ -500,7 +500,7 @@ bool ge_is_on_curve(frameItem* dst, frameItem src, const txEnv* env) {
   return true;
 }
 
-bool scale(frameItem* dst, frameItem src, const txEnv* env) {
+bool off_curve_scale(frameItem* dst, frameItem src, const txEnv* env) {
```

Comment:
Before, the "safe" versions were safe and the normal unprefixed versions were unsafe. With this PR, the "unsafe" versions are unsafe and the normal unprefixed versions are safe. This encourages the use of the safe versions. Is this correct?